packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -7,26 +7,40 @@ import { strict as assert } from "assert";
 
 import { MockHandle } from "@fluidframework/test-runtime-utils";
 
-import { EmptyKey, type FieldKey, type MapTree } from "../../core/index.js";
-import { SchemaBuilder, leaf } from "../../domains/index.js";
-import { FieldKinds, SchemaBuilderBase } from "../../feature-libraries/index.js";
+import {
+	EmptyKey,
+	type FieldKey,
+	type MapTree,
+	type TreeNodeSchemaIdentifier,
+} from "../../core/index.js";
+
+// import { FieldKinds, SchemaBuilderBase } from "../../feature-libraries/index.js";
 // eslint-disable-next-line import/no-internal-modules
-import { InsertableContent } from "../../simple-tree/proxies.js";
+import type { InsertableContent } from "../../simple-tree/proxies.js";
+import {
+	SchemaFactory,
+	// eslint-disable-next-line import/no-internal-modules
```

Comment:
since this is not a test for SchemaFactory, importing SchemaFactory from the simple-tree/index makes more sense in my opinion.